grants/2022_CZI_EOSS5/full_application.md

```diff
@@ -0,0 +1,197 @@
+# scikit-image EOSS5 full application
+
+# Proposal details
+
+## 0. 
```

Comment:
Is this heading necessary or intentional?

Reply:
No, it's not but it is the way our draft was made. :shrug: I'm assuming it corresponds to a section in the application form that wasn't prepared in this document? As this repository is an archive, I don't mind committing the original document without much cleanup.